src/redux/reducers/notificationReducer.js

```diff
@@ -0,0 +1,18 @@
+import { NOTIFICATION } from '../actions/actionTypes';
+
+const initialState = {
+  notify: []
+};
+
+
+export default (state = initialState, action = {}) => {
+  switch (action.type) {
+    case NOTIFICATION:
+      return {
+        ...state, notify: action.notify
```

Comment:
We can change `notify` to `notifications` here too

Reply:
Hi @Dubby20 Kindly look into this